client/verta/tests/registry/test_model_dependencies.py

```diff
@@ -0,0 +1,194 @@
+        'pandas.core.frame'
+    ]
+    extracted_modules: List[str] = [
+        f.__name__ for f in md.list_modules_in_function_signature(func)
+    ]
+    assert extracted_modules == expected_modules
+
+
+def test_list_modules_in_function_return_type_hint_unwrapped(dependency_testing_model) -> None:
+    """ Verify that modules used in function return type hints are extracted
+    as expected with no function wrappers.
+    """
+    func: Callable = dependency_testing_model.unwrapped_predict
+    expected_modules = [
+        'json.encoder',
+        'collections',
+        'sklearn.base',
+        'cloudpickle.cloudpickle_fast',
+        'requests.exceptions'
+    ]
+    extracted_modules: List[str] = [
+        f.__name__ for f in md.list_modules_in_function_signature(func)
+    ]
+    assert extracted_modules == expected_modules
+
+
+def test_list_modules_in_function_return_type_hint_nested(dependency_testing_model) -> None:
+    """ Verify that modules used in function return type hints are extracted
+    as expected when nested inside another type construct.
+    """
+    func: Callable = dependency_testing_model.nested_type_hint
+    expected_modules = ['torch']
+    extracted_modules: List[str] = [
+        f.__name__ for f in md.list_modules_in_function_signature(func)
+    ]
+    assert extracted_modules == expected_modules
+
+
+def test_list_modules_in_function_return_type_hint_multiple(dependency_testing_model) -> None:
+    """ Verify that modules used in function return type hints are extracted
+    as expected when multiple return types are specified.
+    """
+    func: Callable = dependency_testing_model.nested_multiple_returns_hint
+    expected_modules = ['urllib3.util.retry', 'PIL']
+    extracted_modules: List[str] = [
+        f.__name__ for f in md.list_modules_in_function_signature(func)
+    ]
+    assert extracted_modules == expected_modules
+
+
+def test_function_local_annotations(dependency_testing_model) -> None:
+    """ Verify that modules used in function local annotations are extracted as
+    expected when the function is wrapped in verify_io.
+    """
+    func: Callable = dependency_testing_model.predict
+    expected_modules = ['spacy.Errors']
+    extracted_modules =  md.list_function_local_annotation_module_names(func)
+    assert extracted_modules == expected_modules
+
+
+def test_function_local_annotations_unwrapped(dependency_testing_model) -> None:
+    """ Verify that modules used in function local annotations are extracted as
+    expected when the function is not wrapped.
+    """
+    func: Callable = dependency_testing_model.unwrapped_predict
+    expected_modules = ['boto3.Session']
+    extracted_modules =  md.list_function_local_annotation_module_names(func)
+    assert extracted_modules == expected_modules
+
+
+def test_module_names_in_class(dependency_testing_model) -> None:
+    """ Verify that all expected module names are extracted as expected.
+    """
+    expected_module_names = [
```

Comment:
Shall we call this

```suggestion
    expected_module = [
```

to match the other test functions?

Reply:
Done